src/compiler/checker.ts

```diff
@@ -1140,32 +1139,32 @@ const enum WideningKind {
 /** @internal */
 export const enum TypeFacts {
     None = 0,
-    TypeofEQString = 1 << 0,      // typeof x === "string"

```

Comment:
Here's one of many places where we lose the columnar formatting, which would be nice to have formatted as in gofmt.